packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+
+## Which Events are Eligible?
+
+During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures. To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `webhook-eligible`:
+
+[https://developer.okta.com/docs/api/resources/event-types/?q=webhook-eligible](/docs/api/resources/event-types/?q=webhook-eligible)
+
+## Requests Sent by Okta
+
+The requests sent from Okta to your external service are HTTPS requests. POST requests are used for the ongoing delivery of events, and a one-time GET request is used for verifying your endpoint.
+
+### One-Time Verification Request
+
+After registering an event hook, but before you can use it, you need to have Okta make a one-time GET verification request to your endpoint, passing your service a verification value that it needs to send back. This serves as a test confirming that you control the endpoint.
+
+This one-time verification request is the only GET request Okta will send to your external service, while the ongoing requests to notify your service of event occurrences will be HTTPS POST requests. Your web service can use the GET versus POST distinction to implement logic to handle this special one-time request.
+
+The way your service needs to handle this one-time verification is as follows: The request from Okta will contain an HTTP header named `X-Okta-Verification-Token`. Your service needs to read the value of that header and return it in the response body, in a JSON object named `verification`, i.e.: `{ "verification" : "value_from_header" }`. Note thate the value comes to you in an HTTP header, but you need to send it back in a JSON object.
+
+### Ongoing Event Delivery
+
+After verification, for ongoing notification of events, Okta sends HTTPS POST requests to your service. The JSON payload of these requests is where Okta provides specific information about events that have occurred.
+
+Information is encapsulated in the JSON payload in the `data.events` object. The `data.events` object is an array, in order to allow sending multiple events in a single POST request. Events that occur within a short time of each other will be amalgamated, with each each element of the array providing information on one event.
+
+ The content of each array element is an object of the [LogEvent](/docs/api/resources/system_log/#example-logevent-object) type. This is the same object that the [System Log API](/docs/api/resources/system_log/) defines for System Log events. See the documentation there for information on the object and its sub-objects.
+
+ Delivery of events is best effort. Delivery may be delayed by network conditions, and, in some cases, multiple requests can arrive at the same time after a delay. 
```

Comment:
I'd suggest adding a couple of things - 
1. Delivery of events is best effort. `Events are delivered at-least once`.
2. `Events may be delivered out-of-order in some cases. Okta recommends using the `published` attribute of the event object for strict ordering, if needed.`

Reply:
Changed this paragraph to:
"Delivery of events is best effort. Events are delivered at least once. If you have more than one event hook configured to deliver the same event, it will only be sent by one of them. Delivery may be delayed by network conditions. In some cases, multiple requests may arrive at the same time after a delay, or events may arrive out of order. To establish ordering, you can use the time stamp contained in the `data.events.published` property of each event."

Reply:
`If you have more than one event hook configured to deliver the same event, it will only be sent by one of them.` -> We're actually fixing this, and allowing sending same event to multiple hooks.

Reply:
Have now removed the statement that said "If you have more than one event hook configured to deliver the same event, it will only be sent by one of them."